index.html

```diff
@@ -4,9 +4,53 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather Report</title>
+    <link rel="stylesheet" href="styles/index.css">
+    <title> Temperate Weather Report</title>
 </head>
 <body>
+        <header>
+            <h1>Temperate</h1>
+            <h4>Realtime Weather:</h4>
+            <p id="city-header">Your City Name Here!</p>
+        </header>
+
+        <section class="divOne">
+            <h2>Weather Garden</h2>
+            <div id="postDiv">
+                <div id="sky-garden">☀️ ☀️ ☀️ ☀️ ☀️ ☀️ ☀️</div>
+                <div id="landscape">🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷</div>
+            </div>
+        </section>
+
+
+        <section class="divTwo">
+            <h3>Thermostat</h3>
+            <div class=".adhDiv">
```

Comment:
You did a good job using the section tag to organize your webpage. I want to share this resource on the flow for what tags to use. Hopefully, it will be helpful on when to use divs and when not to
![html_flowchart](https://user-images.githubusercontent.com/60710670/174439033-fea7c158-e45f-4f5e-bec7-47f73cccf94a.png)
o